src/app/index.ts

```diff
@@ -130,6 +131,7 @@ module.exports = class TheiaExtension extends Base {
                     { value: ExtensionType.HelloWorld, name: 'Hello World' },
                     { value: ExtensionType.Widget, name: 'Widget' },
                     { value: ExtensionType.LabelProvider, name: 'LabelProvider' },
+                    { value: ExtensionType.Backend, name: 'Backend communication' },
```

Comment:
minor: `Backend Communication` isntead of `Backend communication`

Reply:
done